src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+
+  switch (true) {
```

Comment:
you don't need switch-case-statement here, let's use simple if-statements
when you use switch-case-statement alongside boolean — it means something goes wrong